version-2/Readme.markdown

```diff
@@ -0,0 +1,406 @@
+# Datagrid Version 2
+
+Datagrid v1 was released on Sep 19 2013 - more than 10 years ago.
```

Comment:
I suggest extracting this to `UPGRADING_TO_V2.md` and linking at the top of Readme as it’s quite long.